src/Writers/MarkdownWriter.jl

```diff
@@ -23,9 +22,9 @@ end
 
 function render(doc::Documents.Document, settings::Markdown=Markdown())
     copy_assets(doc)
-    mime = Formats.mimetype(:markdown)
+    mime = MIME"text/plain"()
     for (src, page) in doc.internal.pages
-        open(Formats.extension(:markdown, page.build), "w") do io
+        open(string(splitext(page.build)[1], ".md"), "w") do io
```

Comment:
It would keep the code a bit more readable if we would keep the `string(splitext(page.build)[1], ".md")` part in a small separate function. Especially since the same pattern is used in multiple places in this file. We'd only need a version of `Formats.extension` specialized for `.md` though, so :+1: for getting rid of `Formats` and the functions therein.